src/isomorphic/children/flattenChildren.js

```diff
@@ -18,11 +18,7 @@ var warning = require('fbjs/lib/warning');
 
 var ReactComponentTreeHook;
 
-if (
-  typeof process !== 'undefined' &&
-  process.env &&
-  process.env.NODE_ENV === 'test'
-) {
```

Comment:
Please just change this require to be unconditional. It is no longer necessary to do this trickery now. (The original reason for this was related to some Jest behavior that's irrelevant with flat bundles.)